articles/azure/azs-how-create-S2S-VPN-azure-to-azurestack.md

```diff
@@ -0,0 +1,485 @@
+---
+title: How to create a site-to-site connection using the UKCloud Azure Stack portal | UKCloud Ltd
+description: Create a site-to-site (S2S) VPN gateway connection from an on-premises network to UKCloud for Microsoft Azure 
+services: azure-stack
+author: Shaion O'Niel
+reviewer: Daniel Brennand
+lastreviewed: 
+
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Create a VPN connection
+toc_sub3:
+toc_sub4:
+toc_title: Create a site-to-site VPN connection between Azure Stack & Public Azure - portal
```

Comment:
Can we remove anything from this title to make it a bit shorter?
Create a VPN connection between Azure Stack and Public Azure - Portal
Create a site-to-site connection between Azure Stack and Public Azure - Portal
Create a S2S VPN connection between Azure Stack and Public Azure - Portal